

const authVerify=async (req,res,next) => {
    try {
        
    } catch (error) {
        
    }
}

export {authVerify}